charts/spire/charts/spire-server/templates/tornjak-service.yaml

```diff
@@ -3,20 +3,24 @@ apiVersion: v1
 kind: Service
 metadata:
   namespace: {{ include "spire-server.namespace" . }}
-  name: {{ include "spire-tornjak.backend" . }}
-  {{- with .Values.tornjak.service.annotations }}
+  name: {{ include "spire-tornjak.servicename" . }}
+  {{- with .Values.tornjak.config.service.annotations }}
   annotations:
     {{- toYaml . | nindent 8 }}
   {{- end }}
   labels:
     {{- include "spire-server.labels" . | nindent 4 }}
 spec:
-  type: {{ .Values.tornjak.service.type }}
+  type: {{ .Values.tornjak.config.service.type }}
```

Comment:
This nesting into config deviates from the location where all other charts define the k8s service object. Would prefer to keep that where it was.